YD-142 embed message after action

aristotelos opened the pull request.

Bert-R commented on core/src/main/java/nu/yona/server/messaging/service/MessageService.java:
```diff
@@ -209,4 +209,14 @@ public void removeMessagesFromUser(MessageDestinationDTO destination, UUID sentB
 		MessageDestination.getRepository().save(destinationEntity);
 	}
+
+	public static boolean getIsDirectMessage(MessageDTO message)
```
Please follow the common naming pattern: ``isDirectMessage(...)``.

aristotelos replied:
Indeed, done

Bert-R commented on core/src/main/java/nu/yona/server/messaging/service/MessageService.java:
```diff
@@ -209,4 +209,14 @@ public void removeMessagesFromUser(MessageDestinationDTO destination, UUID sentB
+
+	public static boolean getIsDirectMessage(MessageDTO message)
+	{
+		if (message instanceof BuddyConnectRequestMessageDTO)
```
This approach is a little tricky. Over time, we might add more types of direct messages or, even worse, send some type of message through both message boxes. Let's for now stick with it, as it makes the overall code flow mor readable.

Bert-R commented on core/src/main/java/nu/yona/server/messaging/service/DiscloseRequestMessageDTO.java:
```diff
@@ -122,33 +121,39 @@ public MessageActionDTO handleAction(UserDTO actingUser, Message messageEntity,
 		}
 
-		private MessageActionDTO updateResult(UserDTO respondingUser, DiscloseRequestMessage discloseRequestMessageEntity,
+		private MessageActionDTO updateResult(UserDTO actingUser, DiscloseRequestMessage discloseRequestMessageEntity,
```
I know the method name is not changed, but still: This method name is not very logical to me. What about ``buildReturnValue``?

aristotelos replied:
Renamed to more speaking `updateGoalConflictMessageStatus`

Bert-R commented:
Looks good. I added a few minor comments.

aristotelos commented:
Reworked

On Sat, Jan 9, 2016 at 5:02 PM, Bert Roos <notifications@github.com> wrote:

> Looks good. I added a few minor comments.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yonadev/yona-server/pull/80#issuecomment-170256164>.
>

Bert-R commented:
But not committed or not pushed...

aristotelos commented:
Oops, indeed not pushed correctly, done now

Bert-R commented:
Looks good. I'll merge it after the build of the previous one is completed (a second build while one is active already might break the test run, as that replaces the docker images while the tests are active).
